README.md

```diff
@@ -61,6 +61,8 @@ Daily NuGet builds of the project are also available in our MyGet feed:
 
 ### Get started
 
+* [Building, Debugging and Testing on Windows] (https://github.com/dotnet/roslyn/blob/master/docs/contributing/Building,%20Debugging,%20and%20Testing%20on%20Windows.md)
```

Comment:
I don't think you can (other than just having this file different for each branch)